etc/v4l2loopback_setup.sh

```diff
@@ -1,70 +1,71 @@
 #!/bin/bash
 
-if [ "$(whoami)" == "root" ]; then
-	echo "Please do not run this script directly as root. Use your normal development user account."
+set -euo pipefail
+
+if [[ -z "$BASH_VERSION" ]] ; then
+    echo "Please run this script ($0) with bash, or execute it directly" >&2
+    exit 1
+fi
+
+if [[ "$(whoami)" != "root" ]]; then
+	echo "Please do run this script directly as root" >&2
```

Comment:
Looks like there may be some tabs vs spaces issues/inconsistencies in this file now, I'd recommend standardizing on one or the other before merging.

Reply:
Done